.azure-pipelines.yml

```diff
@@ -143,7 +143,9 @@ jobs:
     VCPKG_INSTALL_ROOT: $(Build.SourcesDirectory)\vcpkg
     VCPKG_ARCH: 'x64-windows'
     VCPKG_PACKAGES: 'assimp boost-system boost-filesystem ccd eigen3 fcl'
-    VCPKG_OPTIONAL_PACKAGES: 'bullet3 ode'
+    # utils needs tinyxml2 and boost algorithm/lexical-cast
+    #   and also boost-math to resolve a circular dependency with lexical-cast
+    VCPKG_OPTIONAL_PACKAGES: 'boost-algorithm boost-lexical-cast boost-math bullet3 ode tinyxml2'
```

Comment:
Do you know if `boost-algorithm`, `boost-lexical-cast`, and `boost-math` are the actual boost components so that we can search them as `find_package(Boost COMPONENTS boost-algorithm boost-lexical-cast boost-math)`? Or, these are just separately packaged in vcpkg?

Reply:
I tried searching for them as `OPTIONAL_COMPONENTS` in DARTFindBoost.cmake, but it failed on Ubuntu because it couldn't find libraries matching those names, and these packages are header-only, so I don't think it works to find as a component; it's just a vcpkg thing